src/tools/runk/src/commands/exec.rs

```diff
@@ -0,0 +1,210 @@
+// Copyright 2021-2022 Kata Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::anyhow;
+use anyhow::Result;
```

Comment:
We can merge these imports sentences like  
```
use std::path::{Path, PathBuf, fs::File}; 

use libcontainer::{status::Status, builder::ContainerBuilder, container::ContainerAction, status::get_current_container_state,};
```

And I think it could be better if the std crate imported first then the third party ones

Reply:
Ok, I will merge them. The sequence is default sorted by `cargo fmt` right now. Maybe cargo can automatically group them by adding a fmt configure file.